### PR TITLE
bulid: add app flycast

### PR DESCRIPTION
**Pull request by 15927146721**
增加flycast应用
image-20231017114505638

### CODE REVIEW

File: flycast/linglong.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package:
+  id: io.github.flycast
+  name: deepin-flycast
```
Comment (kamiyadm) [inline flycast/linglong.yaml:3]: name 就写 flycast吧

**Comment by 15927146721** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by kamiyadm** [comment]
目录名称和id保持一直

**Comment by kamiyadm** [comment]
commit message 看起来有很多重复，再改改呢

**Review by kamiyadm — APPROVED** [review]

**Pull request merged**